osu.Game/Screens/Play/Break/BreakInfoLine.cs

```diff
@@ -58,7 +58,7 @@ public BreakInfoLine(LocalisableString name, string prefix = @"")
 
         private void currentValueChanged(ValueChangedEvent<T> e)
         {
-            string newText = prefix + Format(e.NewValue);
+            LocalisableString newText = LocalisableString.Interpolate($"{prefix}{Format(e.NewValue)}");
```

Comment:
Not only is this `prefix` thing not used anywhere right now (it used to be used to prepend `#` to rank), but if we have a virtual `Format()` method, then is another `prefix` thingy on top of it really required? Any prefix could just as well be put inside `Format()`.

I'd much rather clean it up here while we're at it.